best-practices/native/react-native/deployment/ios/README.md

```diff
@@ -1,79 +1,26 @@
-
-### Caveats to running applications on the emulator:
-
-It's important to note that not all functionality will work with the android emulator (this is also true with the iOS emulator). A common example is that the Camera and Voice Recording of the emulator will not respond to any camera that may be connected to your machine. If you need to test photo or video capture in your App, you'll need to use a physical Android device.
-
-## Android Deployment
-
-
-
-### Setting up Google Play
-
-###### In the [Google Play Console](https://play.google.com/apps/publish/ "Google Play Console")
-- Select 'Create Application'
-- Name Your Application
-
-Before you can upload a build to Google Play, You'll need to set up the minimum allowable information about your application. The list of requirements may change, but currently it is:
-
-1. Add a hi-res icon
-	- must be 512x512px 32-bit PNG. This can contain an alpha channel, but since alpha channels (aka transparency layer) are not allowed in iOS store apps, it's best to create an icon with no alpha channel to prevent different icons on android/iOS
-
-2. Add a feature graphic
-	- 1024x500 JPG or PNG (no alpha channel allowed)
-3. Add at least 2 screenshots
-	- No alpha channel allowed
-
-4. Select an application category
-
-5. Add short and long descriptions of your application
-
-6. Provide a privacy policy URL
-
-7. Provide a content rating
-	- Google Play offers a series of questions that will automatically provide a content rating for you
-
-### Creating an APK
-
-#### What is an APK?
-The APK is the file that represents your application. It is uploaded to the Google Play Store and is eventually downloaded on the user's phone, letting them run your application. Emulators and dev-mode phones can run unsigned APKs, but you'll need to create a signed APK to distribute your app to users.
-
-#### What is a signed APK?
-A signed APK is an APK that has an attached keystore file. The keystore file contains some additional information about the app and the app uploader. If you're putting the app on the Play Store for the first time, you'll need to create a keystore file. If you're updating an existing app, you'll need the existing keystore before you can update the app on the Google Play Store. **IMPORTANT NOTE:**  Make sure you backup your Keystore after creating it. Once an app is published to the Play Store, it will be impossible to update it if you lose the keystore file or password.
-
-#### Steps to Create a signed APK
-###### In Android Studio:
-- Build -> Generate Signed APK
-- Android Studio will ask for the path to your keystore file and its password. If you don't have one, create one here.
-- Set 'Build Type' to 'release' (make note of the APK Destination Folder)
-- Make sure 'V1' and 'V2' Signature Versions are both checked.
-- Click 'finish'
-- After it's done, the APK will be available in the APK Destination Folder
+1. Open the xcode workspace project.
```

Comment:
Not sure if we need to mention anything about Signing team, background modes (Capabilities)

Reply:
Thanks for the feedback!

I thought signing happens automatically now within xcode? Or are we talking about a different Signing process?

Do we have a good reference on the background modes? Are you talking about the permissions that we had to set up on the app?

Reply:
Based on the current application required permissions like `Bluetooth` or `remote notifications` etc...

Reply:
Here I meant just to mention that with no details as a note.